src/colorizer/utils/url_utils.ts

```diff
@@ -164,11 +174,22 @@ export function loadParamsFromUrl(): UrlParams {
   // This assumes there are no negative timestamps in the dataset
   const timeParam = parseInt(urlParams.get(URL_PARAM_TIME) || "-1", base10Radix);
 
+  const colorRampRawParam = safeDecodeString(urlParams.get(URL_PARAM_COLOR_RAMP));
+  let colorRampParam: string | null = colorRampRawParam;
+  let colorRampReversedParam: boolean = false;
+  //  Color ramps are marked as reversed by adding ! to the end of the key
+  if (colorRampRawParam && colorRampRawParam.charAt(colorRampRawParam.length - 1) === "!") {
```

Comment:
Optional: I might be a tiny bit inclined to have a `const COLOR_RAMP_REVERSED_SUFFIX="!";` defined at the top of the file.